packages/openapi-request-validator/test/tests.ts

```diff
@@ -9,6 +9,11 @@ describe(require('../package.json').name, () => {
     new Sut(initialArguments);
     expect(initialArguments).to.eql(createArguments());
   });
+  it('should accept a logger', () => {
+    /* tslint:disable:no-unused-expression */
+    // @ts-ignore
+    new Sut({ logger: console, schemas: [{}] });
```

Comment:
this test covers the constructor, but changes were made to the `#validate` method.  can you add a test that covers those changes?

Reply:
the only change apart from logging statements is in `getSchemaForMediaType()`:
```
  if (!contentTypeHeader) {
    return;
  }
```

and there's a test that already exercises this conditional (should handle missing content type as missing body)

Reply:
ok.  thanks!